fe/fe-core/src/main/java/com/starrocks/analysis/AddColumnsClause.java

```diff
@@ -39,7 +36,7 @@ public class AddColumnsClause extends AlterTableClause {
 
     private Map<String, String> properties;
     // set in analyze
-    private List<Column> columns;
+    private List<Column> columns = new ArrayList<>();
```

Comment:
I think we could abstract a parent class for all these fix column classes, to reduce the duplicated Code

Reply:
Add a abstract class AlterTableColumnClause to reduce duplicated code.